recipes/tarball.rb

```diff
@@ -24,17 +24,16 @@
 # Require for zk gem
 %w(patch gcc).each do |pkg|
   package pkg do
-    action :nothing
+    action :install
     only_if { node['solrcloud']['install_zk_gem'] }
-  end.run_action(:install)
+  end
 end
 
 chef_gem 'zk' do
-  action :nothing
+  action :install
   only_if { node['solrcloud']['install_zk_gem'] }
-end.run_action(:install)
```

Comment:
chef_gem `zk` needs to be installed during compile (for `chef11`), or this behavior is replaced with some resource attribute in `chef12`?